ultralytics/engine/results.py

```diff
@@ -798,8 +798,8 @@ def summary(self, normalize=False, decimals=5):
             box = (row.xyxyxyxy if is_obb else row.xyxy).squeeze().reshape(-1, 2).tolist()
             xy = {}
             for j, b in enumerate(box):
-                xy[f"x{j + 1}"] = round(b[0] / w, decimals)
-                xy[f"y{j + 1}"] = round(b[1] / h, decimals)
+                xy[f"x{j}"] = round(b[0] / w, decimals)
+                xy[f"y{j}"] = round(b[1] / h, decimals)
```

Comment:
We prefer `x1y1x2y2` as the keys in the dict than `x0y0y1y1`, since we have x1y1x2y2 everywhere in our docstrings represents the xyxy format i.e. https://github.com/ultralytics/ultralytics/blob/50497218c25682458ea35b02dcc5d8a364f34591/ultralytics/utils/ops.py#L394